### DISCUSSION TITLE
Is it possible to reorder script-tags from head in nuxt plugin?

### DISCUSSION THREAD

**Discussion by niklaswolf** [nuxt/nuxt#19165] [Questions]
Hi,
I wonder if it is possible to reorder the script tags/entries from head in a Nuxt plugin?
I need to inject a script before every other script in the head.

In Nuxt2 it was possible in a nuxt plugin to achieve this through `app.head.script.unshift({id: 'script-id', innerHTML: '...'})` inside `beforeNuxtRender`.

See also https://github.com/nuxt/nuxt/issues/19260

**Comment by NisuSan** [comment]
I looking for this feature too. But for css files: change order of Naive UI and Tailwindcss

**Comment by niklaswolf** [comment]
Through [this discussion ](https://github.com/unjs/unhead/discussions/118) I was pointed to https://unhead.harlanzw.com/guide/guides/sorting which explains how to set different priorities for scripts, which should be enough for most of the usecases.

**Reply by NisuSan** [reply]
Oh thanks, looks interesting. Now I try to realize how to use this feature for css, which already attached to the dom by nuxt itself. Coz I use Naive UI as an embedded dependency and Tailwindcss as a module.

nuxt.config.ts
```typescript
import Components from "unplugin-vue-components/vite";
import { NaiveUiResolver } from "unplugin-vue-components/resolvers";

export default defineNuxtConfig({
  typescript: {
    shim: false,
  },
  build: {
    transpile:
      process.env.NODE_ENV === "production"
        ? [
            "naive-ui",
            "vueuc",
            "@css-render/vue3-ssr",
            "@juggle/resize-observer",
          ]
        : ["@juggle/resize-observer"],
  },
  vite: {
    optimizeDeps: {
      include:
        process.env.NODE_ENV === "development"
          ? ["naive-ui", "vueuc", "date-fns-tz/esm/formatInTimeZone"]
          : [],
    },
    plugins: [
      Components({
          resolvers: [NaiveUiResolver()], // Automatically register all components in the `components` directory
      }),
    ],
  },
  modules: [
    '@nuxtjs/tailwindcss'
  ],
});

```
